libr/bp/bp.c

```diff
@@ -1,6 +1,8 @@
 /* radare2 - LGPL - Copyright 2009-2018 - pancake */
 
 #include <r_bp.h>
+#include <r_core.h>
```

Comment:
this is wrong you cant use rcore or rdebug from rbp

Reply:
This was slightly problematic but I added new methods to bypass the issue. Is adding RList to corebind ok? Prefer not to return void *

Reply:
Also, couldn't remove r_debug.h because I need to be aware of RDebugMap, should I add a new method to corebind that checks if the addr is in a map and if that map is valid to bypass this? Is it even an issue if I am not using any methods from debug?